### DISCUSSION TITLE
Record Create Duplicate Rule Header

### DISCUSSION THREAD

**Discussion by dalden1981** [forcedotcom/cli#2343] [Ideas]
I'm not sure if this is more an issue or a feature request...

When using the record create command, there doesn't seem to be a way to use the Duplicate Rule Header (https://developer.salesforce.com/docs/atlas.en-us.api_rest.meta/api_rest/headers_duplicaterules.htm)

It would be nice if there was a parameter for the command that would utilize this.

[a comment by an automated account is not shown]

**Comment by dalden1981** [comment]
> This issue has been linked to a new work item: [W-13024803](https://gus.lightning.force.com/lightning/r/ADM_Work__c/a07EE00001PJZVEYA5/view)

Just curious, is there a way for the public to log in and view those work items?

**Comment by WillieRuemmele** [comment]
Hi @dalden1981, there's no way for the public to access those records
